Assets/HoloToolkit/Utilities/Scripts/Editor/SolverHandlerEditor.cs

```diff
@@ -50,13 +50,13 @@ public override void OnInspectorGUI()
 
             serializedObject.ApplyModifiedProperties();
 
-            if (trackedObjectChanged)
+            if (Application.isPlaying && trackedObjectChanged)
             {
                 solverHandler.TransformTarget = null;
                 solverHandler.AttachToNewTrackedObject();
             }
 
-            if (additionalOffsetChanged)
+            if (Application.isPlaying && additionalOffsetChanged)
```

Comment:
Does changing the offset while the game is running actually work? I though we disabled this field since we don't regenerate the offset transforms once they're set. Does the AdditionalOffset and AdditionaRotation accessors reinitialize the offset transforms?

Reply:
Yep, when the additional offset or additional rotation is changed at runtime, we update the tracked transform accordingly: https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/master/Assets/HoloToolkit/Utilities/Scripts/Solvers/SolverHandler.cs#L69

Reply:
Awesome!